package-overrides/npm/redbox-react@1.1.1.json

```diff
@@ -0,0 +1,9 @@
+{
+  "registry": "npm",
+  "main": "dist/redbox",
```

Comment:
@timbur thanks for sharing. I can confirm that the example works fine if I:

* Remove the `npm:redbox-react` override entirely and install it fresh
* Install `jspm install npm:react-transform-catch-errors` instead of `github:gaearon/react-transform-catch-errors`
* Simplifying the override for `github:timbur/babel-plugin-react-transform` to just `{ main: 'src/index' }` was all that was necessary to make everything work.

In the above I didn't hit on any double interop issue case - if you can share what that is I'd be grateful.